### DISCUSSION TITLE
Force landscape orientation [Android]

### DISCUSSION THREAD

**Discussion by tubbadu** [beeware/toga#1445] [Q&A] [answered]
Hi everyone, I'm trying to force the android applications I'm developing to be displayed in landscape orientation, but on the docs I didn't find anything helpful
Is this possible?
Thanks in advance!

**Comment by t-arn** [comment, marked as answer]
I think, you can define this in AndroidManifest.xml

You will need to customize the file provided by the Android template.

In my apps, I copy the file from the template to a build-resources folder, customize it there and then copy it into the right place when building. This way, my customized file is included in the project repo (the build folder is excluded from the repo)

**Reply by tubbadu** [reply]
this worked perfectly, thank you very much!
I edited the file located at `project_name/android/gradle/Hello World/app/src/main/AndroidManifest.xml` and edited line 13:
`<activity android:name="org.beeware.android.MainActivity"  android:screenOrientation="landscape">`

**Reply by t-arn** [reply]
Yes, exactly. Just don't forget to copy it to some folder that will be included in your repo, or you might lose the file when you re-create the build from scratch.
